matplotcheck/tests/test_points.py

```diff
@@ -0,0 +1,173 @@
+"""Tests for the vector module"""
+import pytest
+import matplotlib.pyplot as plt
+import geopandas as gpd
+from matplotcheck.vector import VectorTester
+import matplotlib
+import numpy as np
+
+matplotlib.use("Agg")
+
+
+@pytest.fixture
+def bad_pd_gdf(pd_gdf):
+    """Create a point geodataframe with slightly wrong values for testing"""
+    return gpd.GeoDataFrame(
+        geometry=gpd.points_from_xy(
+            pd_gdf.geometry.x + 1, pd_gdf.geometry.y + 1
+        )
+    )
+
+
+@pytest.fixture
+def edge_gdf(pd_gdf):
+    """Create a point geodataframe to test edge cases found in vector code"""
+    edge_gdf = pd_gdf.append(
+        gpd.GeoDataFrame(geometry=gpd.points_from_xy([0], [0]))
+    )
+    edge_gdf.reset_index(inplace=True, drop=True)
+    edge_gdf.loc[edge_gdf.index == 5, "attr"] = "Flower"
+    return edge_gdf
+
+
+@pytest.fixture
+def pt_geo_plot(pd_gdf):
+    """Create a geo plot for testing"""
+    _, ax = plt.subplots()
+    size = 0
+    point_symb = {"Tree": "green", "Bush": "brown"}
+
+    for ctype, points in pd_gdf.groupby("attr"):
+        color = point_symb[ctype]
+        label = ctype
+        size += 100
+        points.plot(color=color, ax=ax, label=label, markersize=size)
+
+    ax.legend(title="Legend", loc=(1.1, 0.1))
+
+    axis = plt.gca()
+
+    return VectorTester(axis)
+
+
+@pytest.fixture
+def pt_geo_plot_bad(pd_gdf):
+    """Create a geo plot for testing"""
+    _, ax = plt.subplots()
+
+    pd_gdf.plot(ax=ax)
+
+    axis = plt.gca()
+
+    return VectorTester(axis)
+
+
+@pytest.fixture
+def pt_geo_plot_edge(edge_gdf, two_line_gdf):
+    """Create a point plot for edge case testing"""
```

Comment:
let's clarify what these fixtures are. this is a point plot grouped by attribute. we know grouped features do have specific characteristics. 

Let's also consider more specific expressive names for our fixtures. i'm not sure what geo_plot_edge means. what is the edge case? can you come up with better names please? i'm happy to work with you on this.